### ISSUE TITLE
dataframe with Int64 type upload error

### ISSUE THREAD

**Issue by tonylee2016** [influxdata/influxdb-client-python#99]
https://github.com/influxdata/influxdb-client-python/blob/ab189350afbb3b4863cd4631d6ab80e12a7dc5b8/influxdb_client/client/write/point.py#L112

one column of my dataframe is based on numpy.int64.

isinstance(value, int) will not capture it.

type(value)
Out[4]: numpy.int64
isinstance(value, int)
Out[5]: False

related to this one 

https://github.com/numpy/numpy/issues/2951

which suggest to use isinstance(..., np.integer)

**Comment by bednar** [comment]
Hi @tonylee2016,

thanks for using our client.

The PR #97 covers your requirements. As a workaround you could use a version from #97 via:


```bash
pip install git+https://github.com/influxdata/influxdb-client-python.git@feat/dataframe-optimalization
```


Regards

**Comment by bednar** [comment]
Hi @tonylee2016,

The issue is fixed in [1.8.0 milestone](https://github.com/influxdata/influxdb-client-python/milestone/10). 

If you would like to use a dev version then install client via:

```pip install git+https://github.com/influxdata/influxdb-client-python.git@master```

Regards
